en/sim_gazebo_classic/README.md

```diff
@@ -470,6 +470,34 @@ Simply enter the following terminal command:
 gst-launch-1.0  -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' \
 ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink fps-update-interval=1000 sync=false
 ```
+You can also customize the uri where the video stream is sent to, by [customizing the sitl_gazebo_classic parameters](jinja_override)
```

Comment:
The URL target here is `jinga_override`. This doesn't appear to be a document - do you mean `video_uri` in the following section?

Reply:
@frederictaillandier ^^^